module-2/lab-hypothesis-testing/your-code/challenge-1.ipynb

```diff
@@ -19,13 +19,14 @@
   },
```

Comment:
-Se trata de dos muestras de una misma población por lo que deberías utilizar el paired test y , por tanto, la función 
```Python
scipy.stats.ttest_rel(a, b, axis=0)
```
- "Hint: are you comparing different populations or the same population?", en la última pregunta dices, "Comparamos diferentes poblaciones", sin embargo , estás utilizando el método para paired. Dependiendo de como se arme el argumentario, esta pregunta podríamos considerarlo tanto t-test como paired test. Si consideras, contraste de medias de muestras independientes de dos poblaciones el método de scipy es scipy.stats.ttest_ind(a, b, axis=0, equal_var=True) en lugar de ttest_rel.